exercises/05-defining-resource-requirements/solution/solution.md

```diff
@@ -45,10 +45,10 @@ Create the Pod and observe the error message.
 
 ```shell
 $ kubectl create -f pod.yaml --namespace=rq-demo
-Error from server (Forbidden): error when creating "pod.yaml": pods "mypod" is forbidden: exceeded quota: app, requested: requests.memory=1G, used: requests.memory=0, limited: requests.memory=500m
+Error from server (Forbidden): error when creating "pod.yaml": pods "mypod" is forbidden: exceeded quota: app, requested: requests.memory=1G, used: requests.memory=0, limited: requests.memory=500Mi
 ```
 
-Lower the memory settings to less than `500m` (e.g. `200m`) and create the Pod.
+Lower the memory settings to less than `500Mi` (e.g. `524M`) and create the Pod.
```

Comment:
Wouldn't it be better to use `Mi` here to stay within the same metric? `M` would work but I think it might throw people off? WDYT?